internal/config/config.go

```diff
@@ -38,6 +38,10 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
```

Comment:
You should mention what the argument does.
You will need two sentences:
general description of the function,
and a sentence about the argument.